Gemfile

```diff
@@ -30,6 +28,7 @@ gem "uglifier", ">= 1.3.0"
 
 group :development, :test do
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+  gem "capybara"
```

Comment:
gem này để mần chi chú

Reply:
gem này giúp mình tương tác với các thành phần mà người dùng nhìn thấy được, một vài method thường dùng như là: visit, fill_in, click_link, click_button,has_text, has_selector...
khi sử dụng capybara thì dùng feature/scenario thay vì describe/it